app/src/androidTest/java/com/github/se/polyfit/ui/screen/IngredientTest.kt

```diff
@@ -86,6 +86,7 @@ class IngredientTest : TestCase() {
           Ingredient("White Asparagus", 10, 10.0, MeasurementUnit.G),
           Ingredient("Corn", 4, 4.0, MeasurementUnit.G),
           Ingredient("Foie Gras", 100, 100.0, MeasurementUnit.G),
+          Ingredient("Asperge", 27, 1.0, MeasurementUnit.OTHER),
```

Comment:
```suggestion
          Ingredient("Aspargus", 27, 1.0, MeasurementUnit.OTHER),
```

why for White Aspargus this is `MeasurementUnit.G` but for Aspargus it's `MeasurementUnit.Other` ?

Reply:
I wanted to test the Measurement Other. It doesn't display the same way.